### PR TITLE
Update jquery-timeago auto-update config to add locale files

### PR DESCRIPTION
**Pull request by tholu**
Add locales to fileMap

Pull request for issue: #12242 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/rmm5t/jquery-timeago/
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 126
   - Count of stars: 3685
   - Count of forks:  681

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by tholu** [comment]
@sufuf3 Thanks for the extensive review and your suggestions! I created the PR via Github, which was easier for me with the current repo size. Feel free to modify the commit and/or add the other versions in a second commit, very much appreciated!

**Comment by PeterDaveHello** [comment]
@extend1994 can you also help review this PR?

**Comment by tholu** [comment]
@sufuf3 Can you fix this? Thanks!

**Comment by sufuf3** [comment]
@tholu @PeterDaveHello I updated the commit message.
Not sure is it good enough.
@extend1994 Would you help review this PR, please?
Many thanks.

File: ajax/libs/jquery-timeago/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
       {
         "basePath": "",
         "files": [
-          "jquery.timeago.js"
+          "jquery.timeago.js",
```
Comment (extend1994) [inline ajax/libs/jquery-timeago/package.json]: Would `jquery.timeago*` be better (more flexible)?

File: ajax/libs/jquery-timeago/package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
...
+          "locales/*.js"
```
Comment (extend1994) [inline ajax/libs/jquery-timeago/package.json]: Would `locales/jquery.timeago*` be better?

**Comment by sufuf3** [comment]
Helped modify the https://github.com/cdnjs/cdnjs/pull/12243#pullrequestreview-90785710 request.
@extend1994 Please help me review this PR, thanks.

**Pull request merged**